### PR TITLE
Added the functionality to select the multiple person on person property

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
#### Summary
This PR will add the functionality to select multiple person on person property.

#### Ticket Link
Fixes https://github.com/mattermost/focalboard/issues/2413

### CODE REVIEW

**Comment by jespino** [comment]
Hi @Rajat-Dabade I saw your PR and started debating with @wuwinson. I think is better to have 2 different fields (multiperson and person) instead of wand that do both (it can be under the hood, one that do both, but I want to have the ability to restrict the number of persons to one for certain fields, like assigned to). Lets leave this on pause until we decide what to do there.

**Comment by Rajat-Dabade** [comment]
@jespino, Sure thing

File: webapp/src/components/properties/user/user.tsx [hunk 1]
```diff
@@ -95,12 +100,15 @@ const UserProperty = (props: Props): JSX.Element => {
             placeholder={'Empty'}
             getOptionLabel={(o: IUser) => o.username}
             getOptionValue={(a: IUser) => a.id}
-            value={boardUsersById[props.value] || null}
+            value={users}
             onChange={(item, action) => {
+                console.log(action)
```
Comment (wiggin77) [inline webapp/src/components/properties/user/user.tsx:105]: Is this useful without any context?

Comment (Rajat-Dabade) [reply webapp/src/components/properties/user/user.tsx:105]: I forgot to remove it. Nice catch @wiggin77. Thank you.

**Comment by wuwinson** [comment]
Let's hold off on this feature until v7.0 or later. Added a [card with specs](https://community.mattermost.com/boards/workspace/qgsck6cts3fwpqwyjiupjm5cde/b4ctmm917q3rzdfi451qtuxjzcw/vuswsftt1rb873datosmmbtwkpa/cf9zitq5bbjghz8cq54etmxmpcy) for us to review and discuss. We'll want to address some consistency issues with the current Multiselect property as well.

**Comment by LimitlessDonald** [comment]
Thanks for all your work guys .  @wuwinson would the update include a notification sent to the user when the person is added to the person property?

**Comment by wiggin77** [comment]
@Rajat-Dabade is this PR still in progress?

**Comment by Rajat-Dabade** [comment]
@wiggin77, this PR is on hold due to [Refactor of properties](https://github.com/mattermost/focalboard/pull/3356) PR, once the PR is merged, I can resume on this.

**Comment by Rajat-Dabade** [comment]
This is been done using this PR https://github.com/mattermost/focalboard/pull/3741, so closing this issue.

**Pull request closed without merging**